app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -86,6 +92,12 @@ class RoutePresenter(
         when (action) {
             is RouteAction.Welcome -> navigateToFragment(action, R.id.fragment_welcome)
             is RouteAction.Login -> navigateToFragment(action, R.id.fragment_fxa_login)
+            is RouteAction.Onboarding.SkipOnboarding -> navigateToFragment(action, R.id.fragment_item_list)
+            is RouteAction.Onboarding.FingerprintAuth -> navigateToFragment(
+                action,
+                R.id.fragment_fingerprint_onboarding
+            )
+            is RouteAction.Onboarding.Autofill -> navigateToFragment(action, R.id.fragment_autofill_onboarding)
```

Comment:
Is there anywhere that stops devices less than Android O getting to this fragment?